modules/monitoring-viewing-a-list-of-available-metrics.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-metrics.adoc
+
+:_content-type: PROCEDURE
+[id="viewing-a-list-of-available-metrics_{context}"]
+= Viewing a list of available metrics
+
+As a cluster administrator or as a user with view permissions for all projects, you can view a list of metrics available in a cluster and output the list in JSON format.
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+* You have obtained the host URL for the {product-title} route.
+* You have access to the cluster as a user with the `cluster-monitoring-view` role or have obtained a bearer token with `get` permission on the `namespaces` resource.
++
+[NOTE]
+====
+You can only use bearer token authentication to access the Thanos Querier API route.
+====
+
+
+.Procedure
+
+. Retrieve the bearer token:
++
+[source,terminal]
+----
+$ token=`oc whoami -t`
+----
+
+. Retrieve a list of metrics in JSON format from the Thanos Querier API route.
++
+[source,terminal]
+----
+$ curl -k -H "Authorization: Bearer <token>" \
+ https://thanos-querier-openshift-monitoring.apps.<cluster_domain>/api/v1/metadata \
+ | jq '.data' <1>
```

Comment:
piping to `jq` isn't strictly required IMHO.

Reply:
Removed the `jq` portion of the command. We aren't supposed to reference utilities like `jq` in our sample commands.